### PR TITLE
adding the car interface

### PR DESCRIPTION
**Pull request by liampaull**

### CODE REVIEW

File: init_sd_card/stacks/daffy/DT18_03_roscore.yaml [hunk 1]
```diff
@@ -2,7 +2,17 @@ version: '3'
...
+    image: duckietown/dt-duckiebot-interface:daffy
+    restart: unless-stopped
+    network_mode: host
+    privileged: true
+    environment:
+      DISABLE_CONTRACTS: 1
+    volumes:
+    - /data:/data
+
+  duckiebot-interface:
```
Comment (afdaniele) [inline init_sd_card/stacks/daffy/DT18_03_roscore.yaml]: This should be `car-interface`

File: init_sd_card/stacks/daffy/DT18_03_roscore.yaml [hunk 2]
```diff
@@ -2,7 +2,17 @@ version: '3'
...
+    image: duckietown/dt-car-interface:daffy
     restart: unless-stopped
     network_mode: host
     privileged: true
```
Comment (afdaniele) [inline init_sd_card/stacks/daffy/DT18_03_roscore.yaml]: We should still mount `/data`, that is where `dagu_car` takes the calibration from, right?

File: init_sd_card/stacks/daffy/DT18_03_roscore.yaml (hunk 1 above)
Comment (liampaull) [reply init_sd_card/stacks/daffy/DT18_03_roscore.yaml]: good catch thanks

File: init_sd_card/stacks/daffy/DT18_03_roscore.yaml (hunk 2 above)
Comment (liampaull) [reply init_sd_card/stacks/daffy/DT18_03_roscore.yaml]: it is still mounted

Comment (afdaniele) [reply init_sd_card/stacks/daffy/DT18_03_roscore.yaml]: what I meant is that we need 
```
volumes:
    - /data:/data
```
right?

**Pull request merged**